### PR TITLE
Register niko-bot2.dev.is-a.dev

### PR DESCRIPTION
**Pull request by official-notfishvr**
Added `niko-bot2.dev.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

File: domains/niko-bot2.dev.json
Comment (WilliamDavidHarrison) [inline domains/niko-bot2.dev.json:1]: You don't own `dev.is-a.dev` so you cannot register nested subdomains on it.

File: domains/niko-bot.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Sialsdad",
+        "email": "jon111121533@gmail.com"
+    },
+
+    "record": {
+        "MX": ["1"]
```
Comment (WilliamDavidHarrison) [inline domains/niko-bot.json:8]: Please use a valid record.

File: domains/niko-bot2.dev.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Sialsdad",
+        "email": "jon111121533@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "niko"
```
Comment (WilliamDavidHarrison) [inline domains/niko-bot2.dev.json:8]: Please use a valid record.

File: domains/niko_bot.json
Comment (WilliamDavidHarrison) [inline domains/niko_bot.json:1]: You cannot have underscores in your file name.

File: domains/niko_bot.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Sialsdad",
+        "email": "jon111121533@gmail.com"
+    },
+
+    "record": {
+        "A": ["71.239.15.23"]
```
Comment (WilliamDavidHarrison) [inline domains/niko_bot.json:8]: I cannot reach this website, please send a screenshot.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**